test/yml_SDP/CatLibrary.Cat-2.Age.yml

```diff
@@ -27,6 +27,14 @@ members:
   monikers:
   - cat-2.0
   summary: Hint cat's age.
+  remarks: >-
+    ```csharp
+
+    var cat = new Cat();
+
+    var age = cat.Age;  // Will always be 1.
+
+    ```
   metadata: {}
```

Comment:
This spacing for lines was controlled by the existing logic for normalizing in `NormalizeDocsElement`.    To me, it looks wrong when considered against the source and the XML output:

```xml
       <remarks>
          <format langauge="markdown">
            ```csharp
            var cat = new Cat();
            var age = cat.Age;  // Will always be 1.
            ```
            </format>
        </remarks>
```

Reply:
The empty lines are actually added by yaml serializer automatically, it has its own minds and somehow chooses newline style for me, see details in https://yaml-multiline.info/.

Reply:
Thanks for the confirmation; just wanted to make sure that I didn't accidentally introduce weird behavior.